lab_1/dictionary_test.py

```diff
@@ -293,7 +293,7 @@ def test_get_and_left_sort_concordance_ideal(self):
         left_sort = True
 
         expected = ['dog is happy but the cat', 'man is happy the dog is']
```

Comment:
now we expect tokens everywhere from get_concordance till here, like [['dog', 'is'], ['cat', 'is']]